src/bounce_solution/action_calculation.cpp

```diff
@@ -474,7 +474,7 @@ std::vector<double> BounceActionInt::ExactSolutionLin(double l0,
       ss << rho_up << "\t" << l << "\t" << dVdl << "\t" << d2Vdl2 << "\n";
       BSMPT::Logger::Write(BSMPT::LoggingLevel::BounceDetailed, ss.str());
       ss.str(std::string());
-      if (l0 == l0 + (l - l0) / 10.)
+      if (abs((l - l0) / Spline.L) < 1e-10)
```

Comment:
Isn't this now a different check then before? You have `Spline.L` instead of `l0` now

Reply:
Before I trust truncation so that ```l-l0``` was so small compared to ```l0``` so that it vanished in the sum. 
Now I impose that ```l-l0``` is small compared to ```Spline.L```. I basically used your suggestion of having a defined threshold but added ```Spline.L``` to make it scale invariant.